### PR TITLE
Improved the events list table

### LINKED ISSUE #165: Table in list of all events goes out of the wrapper
screen shot 2016-01-05 at 4 23 20 pm

The problem is even worse on smaller screens.

screen shot 2016-01-05 at 4 25 45 pm

### PR DESCRIPTION
**Pull request by niranjan94**
#### What does this PR do?
- Implements [DataTables](https://datatables.net) for displaying the list of events and thereby fixes #165 .
- The edit and buttons are styled and displayed as a button group
- The old sorting and searching related code has been removed since [DataTables](https://datatables.net) takes care of it
- The event logo is displayed as an image instead of URL
- The event URL is a clickable hyperlink
- ellipsizing of the event name, event location name has been disabled

#### Related issue
Issue #165 opened by @shiftsayan on 5<sup>th</sup> January 2016

#### Screenshots
**Before:**
capture2

**After:**
capture

#### How was this tested
- Manually in a browser

### CODE REVIEW

**Comment by niranjan94** [comment]
@leto please review ... :smile:

File: open_event/views/admin/models_views/event.py [hunk 1]
```diff
@@ -45,12 +45,6 @@ class EventView(ModelView):
                    'slogan',
                    'url')
 
-    column_formatters = {
-        'name': Formatter.column_formatter,
-        'location_name': Formatter.column_formatter,
-        'logo': Formatter.column_formatter
```
Comment (niranjan94) [inline open_event/views/admin/models_views/event.py]: I felt this was unnecessary. And the formatter was causing the `logo` URL to be truncated.

File: open_event/static/admin/css/open-event.css [hunk 2]
```diff
@@ -15,6 +15,10 @@
     margin-left: 20%;
     margin-right: 20%
 }
+.table-btn {
+    margin-top: 0;
+    margin-bottom: 0;
+}
 
 ul.form-control {
```
Comment (juslee) [inline open_event/static/admin/css/open-event.css]: Codacy Issue found: [Element (ul.form-control) is overqualified, just use .form-control without element name.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2406576949/issues/source?bid=3233864&fileBranchId=3233884#l23)

**Comment by niranjan94** [comment]
@juslee please review ...

File: open_event/static/admin/css/open-event.css [hunk 3]
```diff
@@ -14,4 +14,12 @@
 .alert-info{
     margin-left: 20%;
     margin-right: 20%
+}
+.table-btn {
+    margin-top: 0;
+    margin-bottom: 0;
+}
+
+ul.form-control {
```
Comment (juslee) [inline open_event/static/admin/css/open-event.css]: Codacy Issue found: [Element (ul.form-control) is overqualified, just use .form-control without element name.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2511700804/issues/source?bid=3233864&fileBranchId=3233884#l23)

**Pull request closed without merging**